RoktDemo.xcodeproj/project.pbxproj

```diff
@@ -432,6 +430,7 @@
 				2574716B261EB5C300B8C0C7 /* NavigationBarTransparent.swift */,
 				25747170261EB5D200B8C0C7 /* NavigationBarBlack.swift */,
 				2570F33B2627B38400B5B52E /* NavigationBarBlackWithButton.swift */,
+				2542B4A92794F2F800863F83 /* NaviigationBarWhite.swift */,
```

Comment:
small typo in navigation (two i's) but apart from that cosmetic thing, all looks good!

Reply:
Great catch. it is fixed now